Sources/Plasma/FeatureLib/pfConsole/pfConsole.cpp

```diff
@@ -332,10 +332,10 @@ bool    pfConsole::MsgReceive( plMessage *msg )
             }
         }
         else if( cmd->GetCmd() == plConsoleMsg::kAddLine )
-            IAddParagraph( cmd->GetString() );
+            IAddParagraph( cmd->GetString().c_str() );
         else if( cmd->GetCmd() == plConsoleMsg::kExecuteLine )
         {
-            if( !fEngine->RunCommand( (char *)cmd->GetString(), IAddLineCallback ) )
+            if( !fEngine->RunCommand( const_cast<char*>(cmd->GetString().c_str()), IAddLineCallback ) )
```

Comment:
The console will (destructively) `strtok` this, which violate's `ST::string`'s immutability...  It would be better to use an `ST::char_buffer`